addons/tmf_loadouts/loadouts/US_USMC_2010_wdl.hpp

```diff
@@ -0,0 +1,563 @@
+	};
+};
+
+class sp : r {
+	displayName = "Spotter";
+	backPack[] = {"CUP_B_AlicePack_Khaki"};
+	backpackItems[] = {
+			LIST_10("CUP_5Rnd_762x51_M24")
+	};
+	items[] += {
+		"ACE_MapTools",
+		"ACE_RangeCard",
+		"ACE_Kestrel4500",
+		"ACE_ATragMX"
+	};
+	linkedItems[] += {
+		"ACE_Yardage450",
+		"ItemGPS"
+	};
+};
+
+class vg : smg {
+	displayName = "Vehicle Gunner";
+	uniform[] = {"SP_0000_Standard_Coverall_Tan"};
+	vest[] = {"CUP_V_B_Eagle_SPC_Crew"};
+	backpack[] = {};
+	headgear[] = {"SP_CVCHelmet_Tan"};
+	magazines[] = {
+		LIST_3("CUP_30Rnd_9x19_MP5"),
+		"SmokeShellPurple"
+	};
+};
+
+class vc : vg {
+	displayName = "Vehicle Commander";
+	linkedItems[] += {
+		"Binocular",
+		"ItemGPS"
+	};
+};
+
+class vd : vg {
+	displayName = "Vehicle Driver";
+	traits[] = {"engineer"};
+	backpack[] = {"CUP_B_AssaultPack_Coyote"};
+	linkedItems[] += {"ItemGPS"};
+	backpackItems[] = {"ToolKit"};
+};
+
+class hc : smg {
+	displayName = "Helicopter Crew";
+	uniform[] = {"FIR_Fighter_Pilot_USN_Nomex4"};
+	vest[] = {"FIR_pilot_vest"};
+	backpack[] = {};
+	headgear[] = {
+		"H_CrewHelmetHeli_B",
+		"FIR_HGU56P_Skull",
+		"FIR_HGU56P_Skull_Death"
+	};
+	items[] += {"ACE_MapTools"};
+	magazines[] = {
+		LIST_3("CUP_30Rnd_9x19_MP5"),
+		"SmokeShellPurple"
+	};
+};
+
+class hp : hc {
+	displayName = "Helicopter Pilot";
+	uniform[] = {"FIR_Fighter_Pilot_USN_Nomex4"};
+	linkedItems[] += {"ItemGPS"};
+};
+
+class hcc : hc {
+	displayName = "Helicopter Crew Chief";
+	traits[] = {"engineer"};
+	backpack[] = {"B_AssaultPack_mcamo"};
+	linkedItems[] += {"ItemGPS"};
+	backpackItems[] = {"ToolKit"};
+};
+
+class jp : smg {
+	displayName = "Jet pilot";
+	headgear[] = {"FIR_JHMCS"};
+	hmd[] = {};
+	vest[] = {"FIR_pilot_vest"};
+	traits[] = {"engineer"};
+	uniform[] = {"FIR_Fighter_Pilot_Uniform_Flight"};
+	backpack[] = {};
+	linkedItems[] += {"ItemGPS"};
+	magazines[] = {
+		LIST_3("CUP_30Rnd_9x19_MP5"),
+		"SmokeShellPurple"
+	};
+};
+
+class eng : car {
+	displayName = "Combat Engineer (Explosives)";
+	traits[] = {"engineer", "explosiveSpecialist"};
+	backpack[] = {"usm_pack_alice"};
+	sidearmWeapon[] = {"ACE_VMM3"};
+	items[] += {
+		"ACE_wirecutter",
+		"ACE_Clacker"
+	};
+	backpackItems[] = {
+		"ToolKit",
+		LIST_3("DemoCharge_Remote_Mag"),
+		LIST_2("SLAMDirectionalMine_Wire_Mag"),
+		"SatchelCharge_Remote_Mag",
+		"ACE_DefusalKit",
+		"ACE_EntrenchingTool",
+	};
+};
+
+class engm : car {
+	displayName = "Combat Engineer (Mines)";
+	traits[] = {"engineer", "explosiveSpecialist"};
+	backpack[] = {"usm_pack_alice"};
+	items[] += {
+		"ACE_wirecutter",
+		"ACE_DefusalKit",
+		"ACE_EntrenchingTool",
+		"ACE_Clacker"
+	};
+	backpackItems[] = {
+		"ToolKit",
+		LIST_4("APERSBoundingMine_Range_Mag"),
+		LIST_2("APERSTripMine_Wire_Mag"),
+		LIST_2("ACE_FlareTripMine_Mag"),
+		LIST_2("ClaymoreDirectionalMine_Remote_Mag")
+	};
+};
+
+class UAV : car {
+	displayName = "UAV Operator";
+	traits[] = {"UAVHacker"};
+	backpack[] = {"B_UAV_01_backpack_F"};
```

Comment:
Remove backpack

Reply:
The RQ-11 was adopted around 2006-ish. Since we don't actually have that, I'm using the base UAV as a stand-in.